packer/install-dependencies.sh

```diff
@@ -361,19 +361,27 @@ K8S_VERSIONS="
 1.12.7
 1.12.6
 1.11.9
+1.11.9-azs
 1.11.8
+1.11.8-azs
```

Comment:
I think you need to add back your previous changes to `cleanUpContainerImages` to clean up the azure stack image in public cloud environments and vice versa.

Reply:
Quick valudation for the change in `cleanUpContainerImages `

```
KUBERNETES_VERSION="1.11.8"
printf '%s\n%s\n' msazurestackdocker/hyperkube-amd64:v1.11.8-azs k8s.gcr.io/hyperkube-amd64:v1.11.8 | \
grep -v "${KUBERNETES_VERSION}$" | \
grep 'hyperkube'

KUBERNETES_VERSION="1.11.8-azs"
printf '%s\n%s\n' msazurestackdocker/hyperkube-amd64:v1.11.8-azs k8s.gcr.io/hyperkube-amd64:v1.11.8 | \
grep -v "${KUBERNETES_VERSION}$" | \
grep 'hyperkube'
```